client/resource_group/controller/model.go

```diff
@@ -237,6 +237,43 @@ func add(custom1 *rmpb.Consumption, custom2 *rmpb.Consumption) {
 	custom1.KvWriteRpcCount += custom2.KvWriteRpcCount
 }
 
+func updateDeltaConsumption(last *rmpb.Consumption, now *rmpb.Consumption) *rmpb.Consumption {
+	delta := &rmpb.Consumption{}
+	if now.RRU >= last.RRU {
+		delta.RRU = now.RRU - last.RRU
+		last.RRU = now.RRU
+	}
+	if now.WRU >= last.WRU {
+		delta.WRU = now.WRU - last.WRU
+		last.WRU = now.WRU
+	}
+	if now.ReadBytes >= last.ReadBytes {
+		delta.ReadBytes = now.ReadBytes - last.ReadBytes
+		last.ReadBytes = now.ReadBytes
+	}
+	if now.WriteBytes >= last.WriteBytes {
```

Comment:
if now with {ReadBytes: 100, WriteBytes: 0}, last is {ReadBytes: 50, WriteBytes: 50}
Maybe make last's parms not consistent, does it make sense?

Reply:
We can assume that the various consumption stored in 'last' is monotonically increasing, and the inconsistency problem is not a big problem, I want to make sure that there are no more statistics RU. If the workload is ongoing and we are counting real consumption, then this discrepancy will be quickly eliminated as well.